pkg/cmd/cmd_add.go

```diff
@@ -257,3 +266,13 @@ func parseOciPkgNameAndVersion(s string) (string, string, error) {
 
 	return parts[0], parts[1], nil
 }
+
+func checkGitTagConsistency(kclPkg *pkg.KclPkg, gitTag string) *reporter.KpmEvent {
```

Comment:
What needs to be checked here is whether the version of the package is consistent with the version of the remote package.

Reply:
`kclPkg.GetPkgVersion()` will give the package version and gitTag consists of the remote version, right. Am I missing something here?

Reply:
cc @zong-zhe

Reply:
I believe this issue should display both the git tag/commit/branch information and the module version information in the `kcl.mod` file, e.g., `k8s = {git = "xxx", tag = "yyy", version = "zzz"}`, rather than just checking for inconsistency.